components/FormField.tsx

```
@@ -0,0 +1,46 @@
+import React from "react";
+
+const FormField = ({
+  id,
+  label,
+  value,
+  onChange,
+  placeholder,
+  type = "text",
```

ESLint

[error] 'type' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)